pkg/util/clientauthorizer/admin.go

```diff
@@ -17,10 +17,6 @@ import (
 // contain a valid client certificate signed by `caBundlePath` and
 // the client certificate's CommonName equals `clientCertCommonName`.
 func NewAdmin(log *logrus.Entry, caBundlePath, clientCertCommonName string) (ClientAuthorizer, error) {
-	if clientCertCommonName == "" {
```

Comment:
make clear what the code path is that causes this to "fail" closed when no configuration is provided